code/__DEFINES/economy.dm

```diff
@@ -0,0 +1,11 @@
+#define SPECIAL_CARGO	1
+#define SPECIAL_SCI		2
+#define SPECIAL_MED		3
+#define SPECIAL_SEC		4
+
+#define SUCCESS						0 //yes this is weird i know
+#define ERROR_NOT_ENOUGH_MONEY		1
+#define ERROR_NONEXISTANT_ACC		2
```

Comment:
As per the comment in my suggested code in my previous code comment (that sounded confusing, wow), why is this not `0` or `null`?